.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,22 @@
+
+name: Pull request
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: mcr.microsoft.com/vscode/devcontainers/python:0-3.10-bullseye
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: | 
+          cd app/backend
+          pip install -r requirements.txt 
+
+      - name: Lint python
+        run: pylint --disable=C app/
```

Comment:
Does the image `vscode/devcontainers/python` already contain pylint?

Reply:
Yup, it does

Reply:
You can see this from the succeeded pipelines pylint output:
![image](https://user-images.githubusercontent.com/12761692/180012968-97f7bc0f-9cdd-48a4-bc84-bc4a7cbd7c00.png)